### DISCUSSION TITLE
Table (with expandable rows) with fixed header and vertical scroll?

### DISCUSSION THREAD

**Discussion by SanjaySankar** [TanStack/table#2216] [General]
I'm working on a project and I need a **table with** :  

**1. 2 fixed headers, 
2. expandable rows and
3. a vertical scroll on the body.**

I have done the expandable rows part but can't find a way to fix the header and height without breaking the component.

**This is the codesandbox link for what I have now : https://codesandbox.io/s/vibrant-napier-h58np**

**Comment by vijay-dadhich09** [comment]
Hi please let me know if you find any solution?

**Comment by abhiGunjan** [comment]
Header can be fixed with vertical scroll by adding following code in `<ReactTable/>` component :
``
style={{
          maxHeight: "600px",
          overflowY: "auto",
          overflowX: "auto",
        }}
``
